_main.sh

```diff
@@ -87,15 +95,7 @@ function runInBuilder() {
     --workdir /app \
     --env GRADLE_USER_HOME=/gradle \
     --env LOCAL_USER_ID="$USER_ID" \
-    --env BINTRAY_USER="$BINTRAY_USER" \
-    --env BINTRAY_API_KEY="$BINTRAY_API_KEY" \
     --env BINTRAY_GPG_PASSPHRASE="$BINTRAY_GPG_PASSPHRASE" \
```

Comment:
Is env used by bintray plugin? I decided to keep it because it doesn't influence our configuration